test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -211,6 +213,116 @@ public void CreateDisplayMetadata_DisplayAttribute_DescriptionFromResources()
             Assert.Equal("description from resources", context.DisplayMetadata.Description());
         }
 
+        [Fact]
+        public void CreateDisplayMetadata_DisplayAttribute_NullLocalizer()
```

Comment:
On closer examination @dougbu is correct, CreateDisplayMetadata_SimpleAttributes does cover this case, will remove.